### DISCUSSION TITLE
Difference between Allure.NUnit and NUnit.Allure packages

### DISCUSSION THREAD

**Discussion by SOFT-VG** [allure-framework/allure2#2683] [General Discussion]
I can't generate Allure Report by using command `allure serve ./allure-result`, when using Allure.NUnit package. But I am able when I Use NUnit.Allure package. WHY?
I don't change nothing when switching between packages.
Please help !!!!

**Comment by baev** [comment]
Is `./allure-result` present?

Please make sure you follow the configuration steps described in the docs: https://allurereport.org/docs/nunit/

`NUnit.Allure` has been deprecated and is no longer maintained. `Allure.NUnit` is not backwards compatible with `NUnit.Allure`; some additional configuration changes may be required.
